creme/creme_core/templates/creme_core/templatetags/listview/entity-filter-summary.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n creme_core_tags creme_widgets %}
+
+<summary>{% blocktrans %}Conditions of «{{efilter}}»{% endblocktrans %}</summary>
+<details>
+{% get_efilter_conditions efilter=efilter user=user as vconds %}
+{% if vconds|length > 1 %}
+    <span>{% if efilter.use_or %}{% trans 'Any condition is met:' %}{% else %}{% trans 'All these conditions are met:' %}{% endif %}</span>
+    <ul>
+        {% for vcond in vconds %}<li>{{vcond}}</li>{% endfor %}
+    </ul>
+{% else %}
+    <span>{{vconds.0}}</span>
+{% endif %}
+</details>
```

Comment:
Maybe in  "creme/creme_core/templates/creme_core/frags" because it's not directly/always related to a templatetag

Reply:
ok